lib/plugins/linkedin.py

```diff
@@ -47,16 +49,16 @@ def wrap_text_with_index(self, content):
         return final_lines
 
     def format_content(self, content, mentions, hashtags, images, **kwargs):
+        # the mentions are not linked to anyone!
         mentions = " ".join([f"@{v}" for v in mentions])
-        hashtags = " ".join(
-            [f"#{v}" for v in hashtags]
-        )  # the mentions are not linked to anyone!
+        hashtags = " ".join([f"#{v}" for v in hashtags])
         if len(images) > 20:
             warnings = f"A maximum of 20 images, not {len(images)}, can be included in a single linkedin post."
             images = images[:20]
         else:
             warnings = ""
 
+        content = strip_markdown_formatting(content)
```

Comment:
Don't we need to split the text first like other platforms?
```suggestion
        content = "\n\n\n".join(
            strip_markdown_formatting(p) for p in content.split("\n\n\n")
        )
```

Reply:
not really. The html parser in base.py now adds an extra line break before paragraphs.
Our readme currently says:

> You can also define custom breakpoints for splitting the post by inserting 2 empty lines anywhere in the text. The second empty line will simply be removed for media that don't support content splitting.

and this is exactly what will happen.